src/Dropdown.tsx

```diff
@@ -278,7 +278,10 @@ function Dropdown({
           (e) => {
             if (
               (e.key === 'Tab' && !e.target) ||
-              !menuRef.current!.contains(e.target as HTMLElement)
+              !(
+                menuRef.current &&
+                menuRef.current.contains(e.target as HTMLElement)
```

Comment:
```suggestion
                menuRef.current?.contains(e.target as HTMLElement)
```
maybe? I think that should be equivalent

Reply:
just checked, it works 😄 thanks

Reply:
In this case, do we want to call `onToggle` if the menu ref isn't defined (menu isn't opened)?